packages/l/libcurl/xmake.lua

```diff
@@ -106,3 +106,4 @@ package("libcurl")
     on_test(function (package)
         assert(package:has_cfuncs("curl_version", {includes = "curl/curl.h"}))
     end)
+ 
```

Comment:
remove empty line

Reply:
It was to force trigger package check

Reply:
you can reverse a commit and push force each time